arseni-ustinovich_26.10.2022

Siion42 opened the pull request:
hw 3 - class person, работа с конструктором
hw 4 - struct, расстояние между 2 точками в пр-ве

Siion42 committed:
homework_1_2

ivan-tms commented:
если это домашка на прошлое занятие, то переименую PR и поставь дату занятия на которое оно было (вчерашнее - 24.10.2022)

Siion42 committed:
Changes to be committed:
	deleted:    HW_1/HW_1.csproj
	deleted:    HW_1/HW_1.sln
	deleted:    HW_1/Program.cs
	deleted:    HW_2/HW_2.csproj
	deleted:    HW_2/HW_2.sln
	deleted:    HW_2/Program.cs

Siion42 committed:
less 2 hw

ivan-tms commented on arseni-ustinovich_26.10.2022/HW_4/Program.cs:
```diff
@@ -0,0 +1,33 @@
+
+        }
+        
+        static double distance(XYZ a, XYZ b) => Math.Sqrt(Math.Pow(a._x - b._x, 2) + Math.Pow(a._y - b._y, 2) + Math.Pow(a._z - b._z, 2));
```
Методы должны называться с большой буквы

ivan-tms commented on arseni-ustinovich_26.10.2022/HW_3/Program.cs:
```diff
@@ -0,0 +1,43 @@
+    internal class Program
+    {
+        
+        static void MyOutput(Person A)
```
параметры методов должны именоваться в нижнем регистре

ivan-tms commented on arseni-ustinovich_26.10.2022/HW_3/Program.cs:
```diff
@@ -0,0 +1,43 @@
+        public string _name;
+        public string _surname;
+
+        public Person(int age, string name, string surname)
```
`this` здесь не обязательно, оно нужно было бы, если поля назывались также как и параметры метода

Siion42 committed:
chaged hw

Siion42 commented:
Исправил!
